### PR TITLE
Setup database

### PR DESCRIPTION
**Pull request by rotimi-babalola**
- setup mongoDB database

### CODE REVIEW

File: server/middleware.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import express from 'express';
```
Comment (ghost) [inline server/middleware.js:1]: I like this file. :)

File: server/server.js [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
+import appConfig from './config';
+import setUpMiddWare from './middleware';
 
 const app = express();
 
 // middlewares mounted
-app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
-app.use(cors());
+setUpMiddWare(app);
```
Comment (ghost) [inline server/server.js:10]: Nice

**Review by ghost — APPROVED** [review]
LGTM

File: server/middleware.js (hunk 1 above)
Comment (rotimi-babalola) [reply server/middleware.js:1]: Gracias

**Pull request merged**